### ISSUE TITLE
Define and Implement a Translation Workflow for LearnOSM

### ISSUE THREAD

**Issue by althio** [hotosm/hotosm-project-ideas#13]
**Proposed by:** @althio

**Summary:** 
- Improve translation, easiness to contribute, delivered quality
- Refine, define and document translation workflow (online/offline, GitHub/Transifex/OmegaT, crowd-sourcing/collaborative)
- Possibilities for using non-technical tools
- Study feasibility of _Translation memory_ "shared glossary" with other HOT/OSM projects

**Skills Required:** Computer-aided translation [CAT], documentation writing, basic knowledge of crowd-sourcing and open projects

**Possible Mentors:** @althio

**Notes:** Breaking up #4 Improve LearnOSM documentation
https://github.com/hotosm/learnosm/issues/3
https://github.com/hotosm/learnosm/issues/138
https://github.com/hotosm/learnosm/blob/gh-pages/CONTRIBUTING.md
https://github.com/hotosm/learnosm/commit/9c6a849827180fc9d94d37dbf2e87e322e0fb0a3
https://lists.openstreetmap.org/pipermail/hot/2015-January/007228.html
http://gis.19327.n5.nabble.com/Fwd-Re-LearnOSM-updates-td5831519.html

**Comment by mataharimhairi** [comment]
@althio Thanks again for the help. I've added the labels as requested and will get in touch with @jmarlena, @Nick-Tallguy and @pyrog to see if they could possibly mentor this project!

**Comment by bgirardot** [comment]
I think this one might be combined with issue #14

**Comment by althio** [comment]
@bgirardot
I agree the issues are related. Indeed one item is common (image localization and filing) and must be handled accordingly.

On the other hand I filed them as separate because the required skills are different and I feel the workload is enough.

For a longer assignment or a gifted intern (you know that during the internship, not before) we could proceed with both ideas.

**Comment by althio** [comment]
@mataharimhairi 
Note that @jmarlena is currently our  HOT intern in the outreach program. Maybe she can mentor or co-mentor this time.

This is the one project I would be most interested in mentoring.

**Comment by juhitandon** [comment]
hello , am an NLP enthusiast and pursuing my major in Computational Linguistics , if someone could please give me more insight into theproject . I would also like to know if there is a coding aspect to it.

Thanks

**Comment by bgirardot** [comment]
@althio we had this conversation on IRC, I said I would put it here so you can correct me where I am wrong:

```
<valisha> if could give me an idea whether this project requires coding ? am pursuing my major in Computational Linguistics , so  i ll be able to put that to use
<valisha> oh i ll just check that out
<BlakeGirardot> No, it is not a lot of coding
<BlakeGirardot> It is more reviewing the website now and the software that generates the website (jekyll) and helping to see if there is a better way to organizae it, make use of jekyll and jekyll plugins
<BlakeGirardot> and help create a workflow for translation into multiple languages.
<valisha> ahh i see 
<BlakeGirardot> That is my understanding. I can put that on the github issue and if the person who proposed it can add or correct me they will.
```

**Comment by althio** [comment]
Hi @juhitandon thank you for your enquiry, thanks @bgirardot for taking the time to answer on IRC and here.

About this project: very true, there is not much coding.
What I imagine is more about reviewing existing tools of computer-aided translation [CAT], then define and document the translation workflow for better quality and performance.

**Comment by juhitandon** [comment]
thanks @bgirardot and @althio for the reply.
so how do you think i should get started in reviewing ? can you elaborate a little please
though little what could be the coding part , if at all ?

**Comment by mataharimhairi** [comment]
@juhitandon

Hello and thank you so much for your interest. I think the first step is to really familarise yourself with the LearnOSM website [1], and the Outreachy guidelines [2]. All potential candidates must be able to dedicate a minimum of 40 hrs a week to the program, which will be carried out remotely from the 25th May until 25th August. We would just like to make sure that everyone is aware of this. 

[1] http://learnosm.org/en/ 
[2] https://www.gnome.org/outreachy/

**Comment by juhitandon** [comment]
yes @mataharimhairi thanks :) I have gone through these and wish to contribute complying to these.
How do i get started doing the actual work ?

**Comment by jmarlena** [comment]
Hey @juhitandon! 

Exploring the "feasibility of translation memory for a shared glossary with other HOT/OSM projects" is probably the closest part of this to coding. See issue [#280](https://github.com/hotosm/learnosm/issues/280) for more about the glossary. In general, I think the most code intensive issues for learnosm involve creating or adapting custom plugins or issue [#11](https://github.com/hotosm/HOT-Project-Ideas/issues/11) for the site.

Not sure if they would rather you post your research/proposal here on this Github issue, as a pull request or as an attachment to your application but you can review the current translation workflow and contribution guidelines for the learnosm site by reading [CONTRIBUTING.md](https://github.com/hotosm/learnosm/blob/gh-pages/CONTRIBUTING.md).  Your questions on any part of the process are helpful for all and most welcome.

**Comment by sandraemma03** [comment]
Hello to all, my name is Sandra. Am happy to be here and i would be glad to participate in this project. I love the ideas and it is really interesting. It would be a great pleasure for me to contribute to it.

**Comment by althio** [comment]
Hi and thank you all for your interest.

We will soon come up with details about the application details.
It will likely include a short cover letter and a simple resume so think about it now.
We will also propose preliminary tasks for you to see if the project suits you and if you are able to tackle it.

**How to start?** Please find [_2 proposed tasks_](https://github.com/hotosm/HOT-Project-Ideas/wiki/Translation-Workflow-for-LearnOSM).

**Comment by althio** [comment]
@jmarlena is right about issue #11 being more related to coding. More comments there (#11) soon.

Glossary of used terms (as in #280) is possibly related to #11 (plugins for automation) and remotely related to #13 (translation) but I do not see that as being exactly in the scope here.
Maybe "shared glossary" for translation was confusing, the right term is certainly "translation memory".

**Comment by sandraemma03** [comment]
Hi, I have issues submitting the assignment on the https://github.com/hotosm/HOT-Project-Ideas/wiki/Translation-Workflow-for-LearnOSM page. I don't have permission submitting to the 
mentors-outreachy@hotosm.org. I want to know if submission needs invitation .
Thanks

**Comment by mataharimhairi** [comment]
Hi Sandra, 

Sorry about that. I will look into why you can't submit to the mentors-outreachy@hotosm.org. Thanks for your patience.

**Comment by althio** [comment]
To start with, we would recommend that you familiarise yourselves with the LearnOSM website [1]. If you haven't already done so, please review the general requirements of the Outreachy internship [2] and keep in mind that a minimum of 40 hrs a week must be dedicated to the program, carried out remotely from the 25th May until 25th August.

[1] http://learnosm.org/
[2] https://www.gnome.org/outreachy/

The next step in the application process would be to complete the items listed below and send them to apply-outreachy@hotosm.org:
- A short paragraph explaining why you are applying and what involvement, if you have had any, in open source projects.
- A simple resume showing previous activities and experience. And please feel free to include work that are not necessarily open source related.
- A draft outlining a proposed time schedule of your availability for the project, incorporating the expected 40 hrs per week from the 25th May to the 25th August.
- [The completed tasks as your contribution to the project](https://github.com/hotosm/HOT-Project-Ideas/wiki/Translation-Workflow-for-LearnOSM).

**Comment by sandraemma03** [comment]
@althio I have been trying to submit my sample proposal to the apply-outreachy@hotosm.org but my mail does not go and i have the following messages:
 You might have spelled or formatted the group name incorrectly.
- The owner of the group may have removed this group.
- You may need to join the group before receiving permission to post.
- This group may not be open to posting.

How can i send it?

**Comment by mataharimhairi** [comment]
Dear applicants, @juhitandon, @sandraemma03,  

Please do not forget to submit your interest for the project via the Outreachy website:

https://outreachy.gnome.org

You must submit your proposal by 24th March 2015 19:00UTC. When submitting your application make sure you select "OpenStreetMap" from the organisation list.

Futher edits and changes to your application can still be made after this, but it is vital that this initial deadline is not missed.

**Comment by dobbler123** [comment]
Hello, Although Its late to hop in but I am really intersted in contributing to this org, given that i have good experience in translation studies and good writing skills as well.
I have submitted the application through Outreachy and now willing to contribute and complete the task for a good and complete application.
Kindly guide me through it.

Thanks

**Comment by althio** [comment]
All requests and steps are outlined up here: https://github.com/hotosm/HOT-Project-Ideas/issues/13#issuecomment-78369145
Including link to proposed tasks: https://github.com/hotosm/HOT-Project-Ideas/wiki/Translation-Workflow-for-LearnOSM

Thank you for your application and please submit the required items as soon as possible.

Please do ask specific questions if you need more help.
